src/assets/simp-environment/build/simp-environment.spec

```diff
@@ -215,21 +214,18 @@ fi
 /usr/local/sbin/simp_rpm_helper --rpm_dir=%{prefix} --rpm_section='postun' --rpm_status=$1 --preserve --target_dir='.'
 
 %changelog
+* Fri Jan 06 2017 Trevor Vaughan <tvaughan@onyxpoint.com> - 6.0.0-Alpha
+- Eliminated the 'localusers' capability
+- Changed the name to simp-environment
+- Ripped out all of the legacy materials
+- Upated to use the simp-adapter RPM helper
```

Comment:
Should we include some JIRA tickets in here to give them a place to look to see what 'Ripped out all of the legacy materials' means?